---
layout: project
title: "Microbiome profiling"
contributors: [nkarcher, qrducarmon, sromano, fspringer, spekel, pwoerz]
handle: profiling
status: ongoing
type: dataset

# Optional
website:
grant:
grant_url:
image: /assets/images/projects/profiling.png
tagline: How can we accurately identify and quantify microbes and their gene functions from various sequencing readouts including ones generated from low-biomass samples? 
tags: 

# Data and code
github: 
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

We are interested in identifying and quantifying microbial taxa and gene functions with high accuracy from various sequencing readouts. Together with the [Sunagawa group](https://micro.biol.ethz.ch/research/sunagawa.html) at [ETH Zurich](http://ethz.ch/), we have developed and maintained the [mOTUs](https://motu-tool.org/) software for high-precision taxonomic profiling from shotgun metagenomic and metatranscriptomic data. Leveraging this expertise, we are now also working on microbial profiling methodologies for low-biomass samples addressing the associated challenges of contamination and low signal-to-noise ratio. Increasingly our focus is shifting towards profiling microbial gene functions and pathways from meta-omics data. As a first step in this direction, we recently developed a software tool ([Cayman](/tools/cayman)) for profiling microbial complex carbohydrate metabolism and are in the process of extending our efforts to other relevant areas of secondary metabolism.
