---
layout: project
title: "GECCO"
contributors: [Martin]
handle: gecco
status: complete
type: software

# Optional
website: "https://gecco.embl.de/"
publications: "https://www.biorxiv.org/content/10.1101/2021.05.03.442509v1"
doi: "10.1101/2021.05.03.442509"
image: /assets/images/projects/2021-05-04-gecco_icon.png
tagline: Gene Cluster prediction with Conditional Random Fields
tags: [bioinformatics, BGC-detection, secondary_metabolism]

# Data and code
github: ["https://github.com/zellerlab/GECCO"]
figshare: ["https://figshare.com/s/83d30901a02d4e8f1d69"]
figshare_names: ["Classifiers"]
osf: "https://osf.io/y7gzp/"
---
{% include JB/setup %}



## Abstract
GECCO is a fast and scalable method for identifying putative novel Biosynthetic Gene Clusters (BGCs) in genomic and metagenomic data using Conditional Random Fields (CRFs).
Biosynthetic gene clusters (BGCs) are enticing targets for (meta)genomic mining efforts, as they may encode novel, specialized metabolites with potential uses in medicine and biotechnology. GECCO is a high-precision, scalable method for identifying novel BGCs in (meta)genomic data using conditional random fields (CRFs). Based on an extensive evaluation of de novo BGC prediction, we found GECCO to be more accurate and over 3x faster than a state-of-the-art deep learning approach in 2023. When applied to over 12,000 genomes, GECCO identified nearly twice as many BGCs compared to a rule-based approach, while achieving higher accuracy than other machine learning approaches. Introspection of the GECCO CRF revealed that its predictions rely on protein domains with both known and novel associations to secondary metabolism. The method developed here represents a scalable, interpretable machine learning approach, which can identify BGCs de novo with high precision.

This software is provided under the [GNU General Public License v3.0 or later](https://choosealicense.com/licenses/gpl-3.0/). GECCO is developped by the Zeller Team at the [European Molecular Biology Laboratory](https://www.embl.de/) in Heidelberg and is part of the suite of computational microbiome analysis tools hosted at EMBL.
GECCO is implemented in [Python](https://www.python.org/), and supports all versions from [Python 3.7](https://endoflife.date/python).

![GECCO Flowchart!](/assets/images/projects/202105-04-gecco-flow.jpg "GECCO_flowchart.jpg")
Graphical depiction of the biosynthetic gene cluster (BGC) identification and classification approach developed here and implemented in GECCO (GEne Cluster prediction with COnditional random fields). Briefly, GECCO identifies open reading frames (ORFs) in an assembled prokaryotic (meta)genome (Step 1). Protein domains are annotated in the resulting ORFs using profile hidden Markov models (pHMMs; Step 2). The resulting ordered domain vectors are treated as features, and a conditional random field (CRF) is used to predict whether each feature belongs to a BGC or not (Step 3). Predicted BGCs are classified into one of six major biosynthetic classes as defined in the Minimum Information about a Biosynthetic Gene cluster (MIBiG) database using a Random Forest classifier (Step 4).

******
## GECCO has been mentioned in the following publications:
1. First item
2. Second item
