---
layout: tools
title: "SIAMCAT"
contributors: [Nicolai]
handle: siamcat
status: complete
type: software
project: [siamcat]

# Optional
website: "https://siamcat.embl.de/"
publications: "https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02306-11"
doi: "10.1186/s13059-021-02306-1"
image: /assets/images/tools/2021-03-30-siamcat-icon.png
tagline: Statistical Inference of Associations between Microbial Communities And host phenoTypes
tags: [bioinformatics, metagenomics]

# Data and code
github: ["https://github.com/zellerlab/siamcat"]
figshare: ["https://figshare.com/s/83d30901a02d4e8f1d69"]
figshare_names: ["Classifiers"]
osf: "https://osf.io/y7gzp/"
---
{% include JB/setup %}

## Abstract
The human microbiome is increasingly mined for diagnostic and therapeutic biomarkers using machine learning (ML). However, metagenomics-specific software is scarce, and overoptimistic evaluation and limited cross-study generalization are prevailing issues. To address these, we developed SIAMCAT, a versatile R toolbox for ML-based comparative metagenomics.  
SIAMCAT is a pipeline for Statistical Inference of Associations between Microbial Communities And host phenoTypes.
A primary goal of analyzing microbiome data is to determine changes in community composition that are associated with environmental factors. In particular, linking human microbiome composition to host phenotypes such as diseases has become an area of intense research. For this, robust statistical modeling and biomarker extraction toolkits are crucially needed. SIAMCAT provides a full pipeline supporting data preprocessing, statistical association testing, statistical modeling (LASSO logistic regression) including tools for evaluation and interpretation of these models (such as cross validation, parameter selection, ROC analysis and diagnostic model plots).  

SIAMCAT is developed in the Zeller group and is part of the suite of computational microbiome analysis tools hosted at [EMBL](https://www.embl.org/).  
<img src="/assets/images/tools/2021-03-30-siamcat-associations.png" alt="SIAMCAT_association_testing.jpg" width="1000"/>  

SIAMCAT is distributed under the [GPL-3](https://www.gnu.org/licenses/gpl-3.0.en.html) license

## GECCO has been mentioned in the following publications:
1. [Cross-cohort gut microbiome associations with immune checkpoint inhibitor response in advanced melanoma](https://www.nature.com/articles/s41591-022-01695-5)
2. [A faecal microbiota signature with high specificity for pancreatic cancer](https://gut.bmj.com/content/71/7/1359)
3. [MGnify: the microbiome sequence data analysis resource in 2023](https://academic.oup.com/nar/article/51/D1/D753/6880769)
