---
layout: tools
title: "PyHMMER"
contributors: [mlarralde]
handle: pyhmmer
status: complete
type: software

# Optional
website: "https://pyhmmer.readthedocs.io/"
publications: "https://academic.oup.com/bioinformatics/article/39/5/btad214/7131068"
doi: "10.1093/bioinformatics/btad214"
image: /assets/images/tools/2023-05-04-pyhmmer-icon.png
tagline: Cython bindings and Python interface to HMMER3.
tags: [bioinformatics, HMM, protein_domain]

# Data and code
github: ["https://github.com/althonos/pyhmmer"]
---
{% include JB/setup %}


## Abstract

PyHMMER provides [Python](https://python.org) integration of the popular profile Hidden Markov
Model software [HMMER](https://hmmer.org) via [Cython](https://cython.org)
bindings. This allows the annotation of protein sequences with profile HMMs
and building new ones directly with Python. PyHMMER increases flexibility of
use, allowing creating queries directly from Python code, launching searches,
and obtaining results without I/O, or accessing previously unavailable
statistics like uncorrected P-values. A new parallelization model greatly
improves performance when running multithreaded searches, while producing the
exact same results as HMMER.

## PyHMMER has been used in the following publications:

1. [Accurate de novo identification of biosynthetic gene clusters with GECCO (preprint)](https://www.biorxiv.org/content/10.1101/2021.05.03.442509v1).
2. [Identification of microbial metabolic functional guilds from large genomic datasets](https://www.frontiersin.org/articles/10.3389/fmicb.2023.1197329/full).
3. [Automated model building and protein identification in cryo-EM maps (preprint)](https://doi.org/10.1101/2023.05.16.541002).
4. [Homologous Pairs of Low and High Temperature Originating Proteins Spanning the Known Prokaryotic Universe](https://www.nature.com/articles/s41597-023-02553-w).