---
layout: tools
title: "Cayman"
contributors: [qrducarmon]
handle: cayman
status: complete
type: software
project: [cayman]

# Optional
website: "https://github.com/zellerlab/cayman"
publications: "https://www.biorxiv.org/content/10.1101/2021.05.03.442509v1"
doi: "10.1101/2021.05.03.442509"
grant: 
grant_url: 
tagline: Profiling carbohydrate-active enzymes in metagenomes
tags: [bioinformatics, metagenomics, cazymes]

# Data and code
github: ["https://github.com/zellerlab/cayman"]
figshare: ["https://figshare.com/s/83d30901a02d4e8f1d69"]
figshare_names: ["Classifiers"]
osf: "https://osf.io/y7gzp/"
---
{% include JB/setup %}

## Abstract

Carbohydrate-active enzymes (CAZymes) are crucial for digesting glycans, but bioinformatics tools for CAZyme profiling and interpretation of substrate preferences in microbial community data are lacking. To address this, we developed the CAZyme profiler Cayman. Cayman (Carbohydrate active enzymes profiling of metagenomes) is an easy-to-use command-line profiling tool for profiling CAZyme abundances in metagenomic datasets. It takes metagenomic shotgun reads as input and produces a matrix of CAZyme Reads-Per-Kilobase-Million (RPKM) abundances for your sample. Cayman enables large-scale surveys of the gut microbial CAZyme repertoire across a large number of human gut metagenomes to reveal how it is shaped by lifestyle and disease.  
By providing an integrated manually curated hierarchical substrate annotation scheme, we facilitate the interpretation of the resulting CAZyme profiles by grouping CAZyme families into higher-level, biologically meaningful substrate groups. Thus CAZyme and substrate annotations are transferred via sequence homology.  
We applied these tools on large-scale bacterial gut genome collections and metagenomic datasets demonstrating their utility for pinpointing bacterial species with specific substrate utilisation patterns (e.g. mucin-foraging) and how glycan substrate utilisation can differ across host lifestyles and health states. Cayman is broadly applicable to (meta-)genomic data from other microbial communities.  

![Cayman overview](/assets/images/tools/2024-01-08-cayman-overview.jpg)

Cayman profiles Carbohydrate-Active Enzymes (CAZymes) from metagenomes via quantification of CAZymes through a gene catalogue. CAZyme abundances are then aggregated/grouped at different levels, facilitated by our curated substrate annotations. GMGC: Gut Microbial Gene Catalog, ORFs: Open Reading Frames, PL: Polysaccharide Lyase, GH: Glycoside Hydrolase.
