---
layout: tools
title: "Cayman"
contributors: [qrducarmon, nkarcher]
handle: cayman
status: complete
type: software
projects: [profiling]

# Optional
website:
publications: "https://www.biorxiv.org/content/10.1101/2021.05.03.442509v1"
doi: "10.1101/2021.05.03.442509"
grant: 
grant_url: 
tagline: Profiling carbohydrate-active enzymes in metagenomes
tags: [bioinformatics, metagenomics, cazymes]

# Data and code
github: ["https://github.com/zellerlab/cayman"]
---
{% include JB/setup %}

## Abstract

Carbohydrates are a crucial component of microbial life, and microbes encode an astounding variety of carbohydrate-active enzymes (CAZymes) that are involved in nutrient foraging, microbe-microbe as well as host-microbe interactions. Despite their evident biological importance, tools for determining the genomic carbohydrate-active potential of microbes are lacking and interpretation of results is difficult because substrate annotations are not readily usable. To address this, we developed the CAZyme profiler Cayman (Carbohydrate active enzymes profiling of metagenomes), which facilitates profiling of CAZyme abundances directly from metagenomes. We further provide a manually curated hierarchical substrate annotation scheme that assigns and describes functionality of CAZymes at different granularities. Using these, we were able to pinpoint previously overlooked mucin foragers in the human gut and shed light on CAZyme differences in the human gut in the context of colorectal cancer and traditional life styles. Cayman is broadly applicable to (meta-)genomic data from a variety of microbial communities.

![Cayman overview](/assets/images/tools/2024-01-08-cayman-overview.jpg)

Cayman profiles Carbohydrate-Active Enzymes (CAZymes) from metagenomes via quantification of CAZymes through a gene catalogue. CAZyme abundances are then aggregated/grouped at different levels, facilitated by our curated substrate annotations. GMGC: Gut Microbial Gene Catalog, ORFs: Open Reading Frames, PL: Polysaccharide Lyase, GH: Glycoside Hydrolase.
